### ISSUE TITLE
Select2 4.0.0 beta.3 - dropdown does not follow select2 select box

### ISSUE THREAD

**Issue by smzchaudary** [select2/select2#2983]
@kevin-brown Congrats on the the Beta.3 release.
Our testers are reporting an issue with the dropdown on the select2. Note, this is unlikely related to any changes in Beta.3.

Essentially, we have a resizable text area element above the select2. If the dropdown is open, and the textarea is resized, the select2 box moves around the page, but the dropdown seems to float in place.

I'm not sure if this is captured in another open issue - it might be related to the window resize issue reported earlier.

I have created a [plunker](http://plnkr.co/edit/pFiJs1v0R1KJgNDUj3EC?p=preview) to demonstrate the issue.

Code from plunker:

```html
<!DOCTYPE html>
<html>

  <head>
    <script data-require="jquery@*" data-semver="2.1.3" src="https://code.jquery.com/jquery-2.1.3.min.js"></script>
    <script src="https://select2.github.io/dist/js/select2.full.js"></script>
    <link href="https://select2.github.io/dist/css/select2.min.css" type="text/css" rel="stylesheet" />
  </head>

  <body>
    <textarea rows="4" cols="50">
    </textarea>
  
    <select style="width: 100%">
      <option value="0">one</option>
      <option value="1">two</option>
      <option value="2">three</option>
      <option value="3">four</option>
    </select>
    <script type="text/javascript">
      $('select').select2({
  placeholder: "Select a color",
});
    </script>
  </body>

</html>
```

**Comment by smzchaudary** [comment]
One quick update: It appears to be isolated to the Chrome browser.

**Comment by kevin-brown** [comment]
I've confirmed that this issue doesn't happen in IE or Firefox. Chrome does not trigger the `mousedown` event that we are looking for when it is resized, so Select2 currently has no way of knowing that the size has changed.

The `resize` even that is most likely triggered also does not bubble, so there isn't much that can be done on our side. I'm going to file a bug with Chromium to see what their thoughts are.

**Comment by smzchaudary** [comment]
@kevin-brown @fk Seems the Chrome Ticket has been resolved as of December 2015. I think we can close this.

**Comment by kevin-brown** [comment]
Looks like this finally landed in Chromium 49.0.2587.0 which appears to have become the stable branch a few weeks ago. Closing this off as I can no longer reproduce the issue.
